webapp/src/client/client.js

```diff
@@ -263,68 +264,76 @@ export default class Client {
         };
     };
 
-    doGet = async (url, headers = {}, credentials) => {
-        headers.Accept = 'application/json';
-        let options = {
-            method: 'get',
-            headers,
-        };
-
-        if (url.includes('plugins/' + pluginID)) {
-            options = Client4.getOptions(options);
-        }
-
-        if (credentials) {
-            options.credentials = credentials;
-        }
+    doGet = (url, headers = {}, credentials) => {
+        return async (dispatch, getState) => {
+            const baseUrl = getPluginServerRoute(getState()) + '/' + url;
```

Comment:
We want to make sure we're only adjusting the URL if the call is to the Mattermost server. Is it possible `doGet` or `doPost` is ever called with a URL that is pointing to a Skype server? In this case we would not want to prepend the site URL.

Reply:
This was also brought up in https://github.com/mattermost/mattermost-plugin-skype4business/pull/78#discussion_r669253152

Reply:
Also `baseUrl` seems like the wrong name for this variable, since this is the URL we will ultimately use in the request.

```suggestion
            url = getPluginServerRoute(getState()) + '/' + url;
```